.travis.yml

```diff
@@ -1,70 +1,57 @@
 # Travis can building for Linux and macOS
-matrix:
+language: python
+addons:
+  apt:
+    update: true
+    packages: libxkbcommon-x11-0  # Linux package needed for Qt to work
+  homebrew:
+    update: true
+    packages: freetype
+
+services: xvfb
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+jobs:
+  allow_failures:
+    - python: 3.8
+    - os: osx
```

Comment:
I suppose I understand the motivation to go `allow_failures` on macOS. But I'd rather have CI fail from time to time and remind us of something that needs fixing. Alternatively, we risk shipping something that does not work for a subset of the users because it is not "properly" tested. Feelings?

PS: And I've been bit by such flaky-ness a few times in the past. ;)

Reply:
> I suppose I understand the motivation to go allow_failures on macOS

Not sure I understand it though, is it the tests flakiness?

Reply:
The motivation for __allow_failures__ is explained at:
https://github.com/mu-editor/mu/pull/957#issuecomment-557967312 and https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail

Reply:
But is that a test that only fails sometimes? As right now the macOS build is passing, right?